assets/js/admin/meta-boxes-order.js

```diff
@@ -697,6 +697,7 @@ jQuery( function ( $ ) {
 					$.post( woocommerce_admin_meta_boxes.ajax_url, data, function( response ) {
 						if ( true === response.success ) {
 							wc_meta_boxes_order_items.reload_items();
+							window.location.href = window.location.href;
```

Comment:
@claudiosmweb, Yes I agree with your concern. should I tried with different logic here and push code for review and do PR ?